.github/workflows/release.yml

```diff
@@ -15,10 +15,11 @@ jobs:
         with:
           fetch-depth: 50
       - run: git tag ${{ github.event.inputs.Version }}
-      - uses: notlmn/release-with-changelog@v2
```

Comment:
This should make it use the local version instead of the previously-published version